src/app/[locale]/(default)/members/[member]/page.tsx

```diff
@@ -0,0 +1,90 @@
+import { memberMockData as memberData } from '@/mock-data/member';
+import { unstable_setRequestLocale } from 'next-intl/server';
+import { notFound } from 'next/navigation';
+import { ArrowLeftIcon } from 'lucide-react';
+//import { useRouter } from 'next/navigation';
```

Comment:
```suggestion

```

Reply:
useRouter needs to be imported from '@/lib/locale/navigation' This is because we have a wrapper for locale. I am just commenting so people know